java/memory/pom.xml

```diff
@@ -46,4 +46,37 @@
   <build>
   </build>
 
+  <profiles>
+    <profile>
+      <!-- This profile turns on integration testing. It activates the failsafe plugin and will run any tests
+      with the 'IT' prefix. This should be run in a separate CI build or on developers machines as it potentially
+      uses quite a bit of memory
+      activate teh tests by adding -Pintegration-tests to your maven command line -->
```

Comment:
Some minor nits:
1. teh -> the
2. we should start a new sentence here?

Reply:
Thanks for the careful read through! These have been fixed.